.travis.yml

```diff
@@ -25,6 +25,9 @@ env:
 
   # Google Tag Manager ID – different for production and preview builds
   - GTM_TAG="GTM-53XG2JT"
+  # Production only variable to allow indexing by robots
+  - GET_ENV_VAR=./bin/set-branch-specific-variable.sh
```

Comment:
Travis will only _do something_ with the build output if it's the master branch, so it should be fine for Travis to always just set this to true.